component/vshn_appcat_services.jsonnet

```diff
@@ -110,7 +110,7 @@ local vshn_appcat_service(name, serviceParams) =
                         plans: std.toString(plans),
                         defaultPlan: serviceParams.defaultPlan,
                         quotasEnabled: std.toString(params.services.vshn.quotasEnabled),
-                        isOpenshift: std.toString(isOpenshift),
+                        isOpenshift: if vars.isControlPlane then std.toString(params.services.vshn.isServiceClusterOpenshift) else std.toString(isOpenshift),
```

Comment:
I'm not really happy with this approach, if someone has a better idea, I'm all ears :-)

Reply:
Can't you use the `distribution` fact to make this automagic?

Reply:
Maybe you could move this whole if else clause into `vars` file? Give it a nice explicit name and you are good to go

Reply:
Have you talked with aldebaran? they might be able to help us. These isCluster or isControlePlaneOpenshit - all this smells very bad. It will bite us later.

Reply:
I've moved the whole logic to `vars`. Please check again if this solution works for you